api/tools/docker-compose.yml

```
@@ -9,17 +9,29 @@ services:
       - 27017:27017
     volumes:
       - db-data:/data
+    healthcheck:
+      test: ["CMD", "mongosh", "--eval", "db.adminCommand('ping')"]
+      interval: 10s
+      timeout: 5s
```

```suggestion
      interval: 2s
```
Setting up mongo is pretty quick, so we don't need to make people wait.